files/templates/mbta/login/messages/messages_en.properties

```diff
@@ -31,4 +62,21 @@ resetpassword.info.lowercase=At least one lowercase letter.
 resetpassword.info.number=At least one number.
 resetpassword.info.specialchar=At least one special character.
 
+password.info=Your password must be at least 12 characters long and contain:
+password.info.li1=At least one uppercase letter
+password.info.li2=At least one lowercase letter
+password.info.li3=At least one number
+password.info.li4=At least one special character
```

Comment:
nitpick: from my days working on translations, it might be easier for translators if the names are more descriptive. maybe:
- `password.info.at_least_one_uppercase`
- `password.info.at_least_one_lowercase`
- et cetera